src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+.search-bar {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
```

Comment:
use one-direction approach - only bottom or top margins to avoid margin collapse